[2024-03-11 05:44:33,358] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: meeting_project492.merge_data 2024-03-11T05:44:27.509782+00:00 [queued]>
[2024-03-11 05:44:33,367] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: meeting_project492.merge_data 2024-03-11T05:44:27.509782+00:00 [queued]>
[2024-03-11 05:44:33,368] {taskinstance.py:1095} INFO - 
--------------------------------------------------------------------------------
[2024-03-11 05:44:33,368] {taskinstance.py:1096} INFO - Starting attempt 1 of 1
[2024-03-11 05:44:33,368] {taskinstance.py:1097} INFO - 
--------------------------------------------------------------------------------
[2024-03-11 05:44:33,379] {taskinstance.py:1115} INFO - Executing <Task(PythonOperator): merge_data> on 2024-03-11T05:44:27.509782+00:00
[2024-03-11 05:44:33,384] {standard_task_runner.py:52} INFO - Started process 903 to run task
[2024-03-11 05:44:33,386] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'meeting_project492', 'merge_data', '2024-03-11T05:44:27.509782+00:00', '--job-id', '690', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmplk08uq23', '--error-file', '/tmp/tmpewper6j9']
[2024-03-11 05:44:33,388] {standard_task_runner.py:77} INFO - Job 690: Subtask merge_data
[2024-03-11 05:44:33,421] {logging_mixin.py:109} INFO - Running <TaskInstance: meeting_project492.merge_data 2024-03-11T05:44:27.509782+00:00 [running]> on host 1c0f3566e916
[2024-03-11 05:44:33,468] {taskinstance.py:1254} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=meeting_project492
AIRFLOW_CTX_TASK_ID=merge_data
AIRFLOW_CTX_EXECUTION_DATE=2024-03-11T05:44:27.509782+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-03-11T05:44:27.509782+00:00
[2024-03-11 05:44:33,504] {python.py:151} INFO - Done. Returned value was: None
[2024-03-11 05:44:33,513] {taskinstance.py:1219} INFO - Marking task as SUCCESS. dag_id=meeting_project492, task_id=merge_data, execution_date=20240311T054427, start_date=20240311T054433, end_date=20240311T054433
[2024-03-11 05:44:33,558] {local_task_job.py:151} INFO - Task exited with return code 0
[2024-03-11 05:44:33,579] {local_task_job.py:261} INFO - 0 downstream tasks scheduled from follow-on schedule check
